test/Microsoft.ML.Core.Tests/UnitTests/TestCSharpApi.cs

```diff
@@ -254,7 +254,7 @@ public void TestCrossValidationBinaryMacro()
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.LessThanNetCore30OrNotNetCore))] // netcore3.0 output differs from Baseline
```

Comment:
What are we doing to track getting these tests enabled on .net core 3?

Reply:
i have opened the issue for it https://github.com/dotnet/machinelearning/issues/1096

It contains the list of tests that we fixed and ones still remaining